canonical-web-and-design/jaas-dashboard
# Add UI for viewing and editing charm configuration

## Done

Add a UI for viewing and editing charm configuration.

## QA

Note: each PR from the feature branch has already been reviewed and qa'd but this is for all the marbles!

- View the demo link.
- Navigate to a model and view the configuration for an application.
- See if you can break it.
- Try it out on mobile/mobile views

## Screenshots

![Screen Shot 2021-01-25 at 12 54 35 PM](https://user-images.githubusercontent.com/532033/105752130-83c67600-5f0c-11eb-9d10-19a88f18f09d.png)
